Update simplify-owa-url.md (#3163)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/hybrid-deployment/simplify-owa-url.md

   > [!NOTE]
   > 

>    > - Use http, not https. When HTTPS is used, users may get an URL redirection issue [ AADSTS50011 ]
>    > 

Reviewer: ```suggestion
   > - Use http, not https. If https is used, users may get a URL redirection issue [AADSTS50011].
```
